app/services/event_log/move_runner.rb

```diff
@@ -23,6 +23,7 @@ def call
           move.rejection_reason = event.rejection_reason
           move.cancellation_reason = Move::CANCELLATION_REASON_REJECTED
           move.cancellation_reason_comment = event.cancellation_reason_comment
+          move.rebook.save if event.rebook?
```

Comment:
Keep in mind that the runner can/will run a move's events multiple times. Will this change create a new move record whenever the runner runs the move's event queue? Probably better to only create a new record if there isn't already a rebooked move? And in that case you migt want to add an index on `original_move_id` ?

Reply:
Very good point - now refactored to make `rebook` idempotent, so re-running the `reject` event will not create multiple rebookings.